docs/cloud-nativeness/kubernetes.md

```diff
@@ -208,6 +208,39 @@ If you already have the simple Flow from the first example running on your clust
 kubectl apply -R -f ./k8s_flow
 ```
 
+### Deploy Flow with custom environment variables and secrets
+
+You can customize the environment variables that are available inside runtime, either defined directly or read from a [Kubernetes secret](https://kubernetes.io/docs/concepts/configuration/secret/):
+
+```python
+from jina import Flow
+
+f = (
+    Flow(port=8080)
+    .add(
+        name='indexer',
+        uses='jinaai+docker://jina-ai/ANNLiteIndexer',
+        uses_with={'dim': 512},
+        env={'k1': 'v1', 'k2': 'v2'},
+        env_from_secret={
+            'SECRET_USERNAME': {'name': 'mysecret', 'key': 'username'},
+            'SECRET_PASSWORD': {'name': 'mysecret', 'key': 'password'},
+        },
+    )
+)
+
+f.to_kubernetes_yaml('./k8s_flow', k8s_namespace='custom-namespace')
+```
```

Comment:
can we create 2 examples ?
first one is in python (the current example) and the second one is with flow YAML and using command `jina export ...` command. Use a tab to separate between both examples